holmes/plugins/toolsets/coralogix/utils.py

```diff
@@ -0,0 +1,166 @@
+import json
+import logging
+from datetime import datetime
+from typing import Any, Optional, Dict, List
+
+from pydantic import BaseModel
+
+
+def parse_json_lines(raw_text) -> List[Dict[str, Any]]:
+    """Parses JSON objects from a raw text response."""
+    json_objects = []
+    for line in raw_text.strip().split("\n"):  # Split by newlines
+        try:
+            json_objects.append(json.loads(line))
+        except json.JSONDecodeError:
+            logging.error(f"Failed to decode JSON from line: {line}")
+    return json_objects
+
+
+def format_kubernetes_info(
+    kubernetes: Optional[dict[str, str]], add_namespace_tag: bool, add_pod_tag: bool
+):
+    tags = []
+    if kubernetes:
+        if add_pod_tag and kubernetes.get("pod_name"):
+            tags.append(f'pod_name="{kubernetes.get("pod_name")}"')
+        if add_namespace_tag and kubernetes.get("namespace_name"):
+            tags.append(f'namespace_name="{kubernetes.get("namespace_name")}"')
+    return " ".join(tags)
+
+
+""" improves human readability of logs by indenting logs that span multiple lines.
```

Comment:
Should this be part of the doc string of method `indent_multiline_log_message`?

Reply:
Ah right. It's inside the func in python...